### PR TITLE
Merge lipod install.md to this repo

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Merge the installation.md from Libpod's GitHub repo to this repo.  A link
will be left on the repo side of things to point to this merged file.
We've had a number of edits made to both files, sometimes not at the same
time and we'd a pretty good divergence in only a few months.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@baude @mheon @lsm5 @vrothberg @rhatdan PTAL

File: getting-started/installation.md [hunk 1]
```diff
@@ -228,14 +255,14 @@ sudo apt-get install \
...
   libc6-dev \
   libdevmapper-dev \
   libglib2.0-dev \
   libgpgme-dev \
   libgpg-error-dev \
-  libostree-dev \
   libprotobuf-dev \
-  libprotobuf-c-dev \
+  libprotobuf-c0-dev \
   libseccomp-dev \
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:266]: These two changes to the Debian install were the two I was least comfortable with.  @lsm5 can you verify that they're correct?

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Any more LGTM's?

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**